pajbot/modules/quest.py

```diff
@@ -24,15 +24,15 @@ class QuestModule(BaseModule):
             type="options",
             required=True,
             default="say",
-            options=["say", "whisper", "me", "reply"],
+            options=["say", "whisper", "me", "announce", "reply"],
```

Comment:
Wouldn't be a good idea for this command
```suggestion
            options=["say", "whisper", "me", "reply"],
```